e2e/terraform/main.tf

```diff
@@ -20,5 +21,12 @@ module "provision-infra" {
   nomad_region                           = var.nomad_region
   instance_arch                          = var.instance_arch
   instance_type                          = var.instance_type
-  name                                   = var.name
+  volumes                                = var.volumes
```

Comment:
I suppose these have been leaning on the defaults?

Reply:
Yeah, exactly.